### ISSUE TITLE
Atcommand follow after death

### ISSUE THREAD

**Issue by MrAntares** [rathena/rathena#2424]
* **rAthena Hash**: 

c87dba5

* **Client Date**: 

2012-04-10

* **Server Mode**: 

RE

* **Description of Issue**: When following someone and the followed player dies, the `@follow` stops following, but the `@follow` mode will be still on. If the target player respawns, the `@follow` won't continue following the target player. If we want to initiate follow again, we have to use `@follow` twice: 1 to stop the now not-working follow, 2 to start following again.

  * Expected Result: There is 2 possible fix. 
1: When the target dies the follow should stop, and you should quit the follow mode.
2: When the target dies, the follow mode stays on, and when the target respawns, it continues to follow the target player. This version would be the better for administrators and GMs.

**Comment by Lemongrass3110** [comment]
Please use ticks ` to quote atcommands. You linked the user "follow" 5 times and "following" 1 time as well.
I've edited your post this time. Make sure you quote them correctly next time.

**Comment by MrAntares** [comment]
@Lemongrass3110 Thanks!

**Comment by aleos89** [comment]
I can't reproduce the multiple retypes of the command you are talking about. When a player dies currently it clears all the follow related values. I followed a player multiple times and tried different variations of returning the character to a different map and what not, but I only had to retype the follow command once to continue following the player again.

On your other point, I do agree with following the player even if the player dies. Not sure why it was implemented like this.
